packages/esm-patient-registration-app/src/root.component.tsx

```diff
@@ -46,6 +46,10 @@ export default function Root({ savePatientForm, isOffline }: RootProps) {
                 path=""
                 element={<PatientRegistration savePatientForm={savePatientForm} isOffline={isOffline} />}
               />
+            </Routes>
+          </BrowserRouter>
+          <BrowserRouter basename={`${window['getOpenmrsSpaBase']()}`}>
+            <Routes>
               <Route
                 path="/patient/:patientUuid/edit"
                 element={<PatientRegistration savePatientForm={savePatientForm} isOffline={isOffline} />}
```

Comment:
```tsx
<BrowserRouter basename={window['getOpenmrsSpaBase']()}>
  <Routes>
    <Route
      path="/patient-registration"
      element={<PatientRegistration savePatientForm={savePatientForm} isOffline={isOffline} />}
    />
    <Route
      path="/patient/:patientUuid/edit"
      element={<PatientRegistration savePatientForm={savePatientForm} isOffline={isOffline} />}
    />
  </Routes>
</BrowserRouter>
```

Reply:
Thanks for your feedback, should be done now.